test/units/testsuite-71.sh

```diff
@@ -89,10 +89,47 @@ test_chassis() {
     fi
 }
 
+restore_sysfs_dmi() {
+    umount /sys/class/dmi/id
+    rm -rf /run/systemd/system/systemd-hostnamed.service.d
+    systemctl daemon-reload
+    systemctl restart systemd-hostnamed
+}
+
+test_firmware_date() {
+    trap restore_sysfs_dmi RETURN
+
+    # Ignore /sys being mounted as tmpfs
+    mkdir /run/systemd/system/systemd-hostnamed.service.d/
+    cat > /run/systemd/system/systemd-hostnamed.service.d/override.conf <<- EOF
```

Comment:
We usually do not insert space between `>` and filename, (also maybe between `<<` and `EOF`, though I do not know why you set `-` after `<<`):
```
    cat >/run/systemd/system/systemd-hostnamed.service.d/override.conf <<EOF
```